src/components/metadata/metadata.test.js

```diff
@@ -376,54 +430,64 @@ describe('MetaData', () => {
 
   describe('Plot nodes', () => {
     it('shows the node type as an icon', () => {
-      const wrapper = mount({ nodeId: bullPlotNodeID });
+      const wrapper = mount({
+        nodeId: bullPlotNodeID,
+        mockMetadata: node_plot,
+      });
       expect(rowIcon(wrapper).hasClass('pipeline-node-icon--icon-plotly')).toBe(
         true
       );
     });
 
     it('shows the node name as the title', () => {
-      const wrapper = mount({ nodeId: bullPlotNodeID });
+      const wrapper = mount({
+        nodeId: bullPlotNodeID,
+        mockMetadata: node_plot,
+      });
       expect(textOf(title(wrapper))).toEqual(['Bull']);
     });
 
     it('shows the node type as text', () => {
-      const wrapper = mount({ nodeId: bullPlotNodeID });
+      const wrapper = mount({
+        nodeId: bullPlotNodeID,
+        mockMetadata: node_plot,
+      });
       const row = rowByLabel(wrapper, 'Type:');
       expect(textOf(rowValue(row))).toEqual(['data']);
     });
 
     it('shows the node filepath', () => {
-      const wrapper = mount({ nodeId: bullPlotNodeID });
+      const wrapper = mount({
+        nodeId: bullPlotNodeID,
+        mockMetadata: node_plot,
+      });
       const row = rowByLabel(wrapper, 'File Path:');
       expect(textOf(rowValue(row))).toEqual(['-']);
     });
 
     it('shows the node tags', () => {
-      const wrapper = mount({ nodeId: bullPlotNodeID });
+      const wrapper = mount({
+        nodeId: bullPlotNodeID,
+        mockMetadata: node_plot,
+      });
       const row = rowByLabel(wrapper, 'Tags:');
       expect(textOf(rowValue(row))).toEqual(['Small']);
     });
 
     it('shows the node pipeline', () => {
-      const wrapper = mount({ nodeId: bullPlotNodeID });
+      const wrapper = mount({
+        nodeId: bullPlotNodeID,
+        mockMetadata: node_plot,
+      });
       const row = rowByLabel(wrapper, 'Pipeline:');
       expect(textOf(rowValue(row))).toEqual(['Default']);
     });
 
     describe('shows the plot info', () => {
-      const metadata = getClickedNodeMetaData(
-        prepareState({
-          data: animals,
-          afterLayoutActions: [() => toggleNodeClicked(bullPlotNodeID)],
-        })
-      );
-      metadata.plot = node_plot.plot;
-
-      const wrapper = setup.mount(
-        <MetaData visible={true} metadata={metadata} />
-      );
-
+      const wrapper = mount({
+        nodeId: bullPlotNodeID,
+        mockMetadata: node_plot,
+      });
```

Comment:
with the above proposed setup, I would instead keep the previous setup, as it is much more readable to see that this test is testing the display of a field returned from the metadata.